CONTRIBUTING.md

```diff
@@ -102,19 +182,31 @@ GitHub web notifications using a tool such as [Octobox](http://octobox.io/).
 
 ## Contributing outside this repository
 
-There are many useful ways to contribute to the Coq ecosystem that don't involve the Coq repository.
-
-Tutorials to teach Coq, and especially to teach particular advanced features, would be much appreciated. Some tutorials are listed on the [Coq website](https://coq.inria.fr/documentation). If you would like to add a link to this list, please make a pull request against the Coq website repository at https://github.com/coq/www.
-
-External plugins / libraries contribute to create a successful ecosystem around Coq. If your external development is mature enough, you may consider submitting it for addition to our CI tests. Refer to [`dev/ci/README.md`](/dev/ci/README.md) for more information.
-
-Ask and answer questions on [Stack Exchange](https://stackexchange.com/filters/299857/questions-tagged-coq-on-stackexchange-sites) which has a helpful community of Coq users.
-
-Hang out on the Coq IRC channel, `irc://irc.freenode.net/#coq`, and help answer questions.
-
-[rebase-label]: https://github.com/coq/coq/pulls?utf8=%E2%9C%93&q=is%3Aopen%20is%3Apr%20label%3A%22needs%3A%20rebase%22
-[fixing-label]: https://github.com/coq/coq/pulls?q=is%3Aopen+is%3Apr+label%3A%22needs%3A+fixing%22
-[benchmarking-label]: https://github.com/coq/coq/pulls?q=is%3Aopen+is%3Apr+label%3A%22needs%3A+benchmarking%22
-[testing-label]: https://github.com/coq/coq/pulls?q=is%3Aopen+is%3Apr+label%3A%22needs%3A+testing%22
-
-[jenkins-doc]: https://github.com/coq/coq/wiki/Jenkins-(automated-benchmarking)
+There are many useful ways to contribute to the Coq ecosystem that don't
+involve the Coq repository.
+
+Tutorials to teach Coq, and especially to teach particular advanced features,
+are much appreciated. Some tutorials are listed on the
+[Coq website](https://coq.inria.fr/documentation). If you would like to add
+a link to this list, please make a pull request against the Coq website
+repository at <https://github.com/coq/www>.
+
+External plugins / libraries contribute to create a successful ecosystem
+around Coq. If your external development is mature enough, you may consider
+submitting it for addition to our CI tests. Refer to
+[`dev/ci/README-users.md`](dev/ci/README-users.md) for more information.
+
+Some Coq packages are not maintained by their authors anymore even if they
+were useful (for instance because they changed jobs). The coq-community
+organization is a place for people to take over the maintenance of such
+useful packages. If you want to contribute by becoming a maintainer, you can
+find a list of packages waiting for a maintainer [here](https://github.com/coq-community/manifesto/issues?q=is%3Aissue+is%3Aopen+label%3Amaintainer-wanted).
+You can also propose a package that is not listed. Find out more about
+coq-community in [the manifesto's README](https://github.com/coq-community/manifesto).
+
+Ask and answer questions on our [Discourse forum][], on [Stack Exchange][],
+and on the [Coq IRC channel][].
```

Comment:
This link does not actually show up in the rendered page; maybe github doesn't recognize irc://?